extensions/exchanges/gdax/exchange.go

```diff
@@ -21,6 +22,7 @@ type gdaxExchange struct {
 	BackfillRateLimit int64
 }
 
+// NewGDAXExchange add new exchange to coinbase(gdax)
```

Comment:
We'll probably want to re-phrase this to "returns a new Coinbase (gdax) Exchange interface implementation"